waspc/src/Wasp/Generator/ServerGenerator/OAuthAuthG.hs

```diff
@@ -0,0 +1,119 @@
+module Wasp.Generator.ServerGenerator.OAuthAuthG
```

Comment:
Maybe we should have groupped it as ServerGenerator/Auth/LocalG and ServerGenerator/Auth/OAuthG?